fix: relative distance (#2545)
exercism/problem-specifications: exercises/relative-distance/instructions.md

There's no known relationship between Isla and [Kevin][six-bacons], as there is no connection in the given data.
The degree of separation between Uma and Isla is 1.

~~~~exercism/note

> Isla and Tariq are siblings and have a separation of 1.

Reviewer: There should be a new line here if what follows is a different example
```suggestion
Isla and Tariq are siblings and have a separation of 1.

```

> For a genetic relationship analysis, you wouldn't be related to the person that married the brother of your father.
> This implementation would report a relationship distance of 2 from you to that person.

Reviewer 2: I don't know if that's true, the sibling rule is just a shortcut, not a new relation. In this exercise, there is no notion of marriage, husband and wife are not siblings, so you wouldn't be related to your uncle's wife (unless they have children you can go through, or some weird family business is going on).
I'd simplify this to
```suggestion
Similarly, this implementation would report a relationship distance of 2 from you to your father's brother.
```

Reviewer: Based on the above suggestion, this change is only for consistency reasons (disregard if i'm missing something)
```suggestion
For a genetic relationship analysis, you wouldn't be related to the person that married the brother of your father.
This implementation would report a separation of 2 between you and that person.
```
